pysat/utils/_core.py

```diff
@@ -1,9 +1,8 @@
-from __future__ import print_function
-from __future__ import absolute_import
-
+import datetime as dt
+from importlib import import_module
+from importlib import reload as re_load
```

Comment:
Why the nickname?

Reply:
inherited from the python 2/3 compatibility days.  I could fix it here or we could do a larger scrub later.

Reply:
I thought that might be it.  I'd recommend just having `import importlib` now that we aren't supporting Python 2.

Reply:
And I am a fan of fixing the files as we go, if you're cool with that.

Reply:
fixed here and in test_instrument.py